docs/3.0.0-beta.x/guides/count-graphql.md

```diff
@@ -0,0 +1,79 @@
+# Count with GraphQL
+
+This guide explains how to count data with a GraphQL query.
+
+The count aggregation has some issues and this feature is also not available for Bookshelf (SQL databases).
+
+With this guide we will code our own count query.
+
+Here is the [GraphQL documentation](../plugins/graphql.md#customise-the-graphql-schema) we will use to achieve our goal.
+
+## Setup the application
+
+In this example, we will use a **Restaurant** API.
+
+Make sure your have a **Content Type** with some entries.
+
+## Create schema.grahql file
+
+To be able to add a new custom query (or mutation), we will have to create a `schema.grahql` file in your **Restaurant** API.
```

Comment:
We can create a `schema.graphql.js`  this way editors wont mistake it with a realy grahpql file.